src/d3a/d3a_core/launcher.py

```diff
@@ -37,8 +37,9 @@ def __init__(self,
                  queue=None,
                  max_jobs=None,
                  max_delay_seconds=2):
-        self.queue = queue or Queue(get_simulation_queue_name(), connection=StrictRedis.from_url(
-            REDIS_URL, retry_on_timeout=True))
+
+        self.redis_connection = StrictRedis.from_url(REDIS_URL, retry_on_timeout=True)
+        self.queue = queue or Queue(get_simulation_queue_name(), connection=self.redis_connection)
```

Comment:
How splitting it helps?

Reply:
Nevermind, I think I got it, you intend to minimise number of redis connection?

Reply:
Yes, in order to reuse the same connection.